code-challenges.js

```diff
@@ -40,24 +79,72 @@ var vowelTester1 = "learn"
 // Expected output: 1
 var vowelTester2 = "throw"
 // Expected output: 3
-
-
-
-
-
+//Create a function that iterates each letters
+//have another function intandem to return the index of only the first vowel indexOf()
+//turn string into array of letters
+//for loop that checks AEIOU
+//return index
+var arrayVowel = ["e" , "a" , "o" , "u" , "i"]
+
+const indexr = (string) => {
+  let stringSplit = string.split("")
+  //console.log(stringSplit);
+  for(let i=0; i<stringSplit.length; i++){
+    if (arrayVowel.includes(stringSplit[i]))
+    return i
+  }
+  //return stringSplit
+  //return stringSplit.indexOf(value === "e" && "a" && "i" && "o" && "u")
+  //return stringSplit.indexOf("e" || "a" || "i" || "o" || "u")
+}
+console.log(indexr(vowelTester1));
+console.log(indexr(vowelTester2));
```

Comment:
Great! I'm curious, would there be a way to complete this with a higher-order-function?